miden-lib/asm/kernels/transaction/lib/memory.masm

```diff
@@ -494,7 +500,7 @@ end
 #!
 #! Where:
 #! - ACCT_ROOT is the account root of the last known block.
-export.get_account_db_root
+export.get_acct_db_root
```

Comment:
Because other procedures use `acct` instead of `account` this is one of the things I believe that should be standardised in the kernel conventions. 

How do we call accessors, setters, names of return values, errors, doc comments, etc...

#858